tests/phpunit/class-wc-stripe-order-helper-test.php

```
@@ -165,4 +165,27 @@ public function test_is_stripe_gateway_order(): void {
+
+		$setup_intent = (object) [
+			'id'                   => 'seti_mock',
+			'object'               => 'setup_intent',
+			'payment_method_types' => [ 'card' ],
+		];
+
+		// Returns void on success; a stale-amount read would throw here instead.
+		$this->helper->validate_intent_for_order( $order, $setup_intent );
+
+		$this->assertTrue( true, 'Setup intent validated without reading a missing amount.' );
+	}
```

PHPMD (2.15.0)

[error] The method test_validate_intent_for_order_accepts_setup_intent_without_amount is not named in camelCase. (undefined)

(CamelCaseMethodName)